lib/src/fnc/array.rs

```diff
@@ -105,6 +105,24 @@ pub fn union(arrays: (Value, Value)) -> Result<Value, Error> {
 	})
 }
 
+pub fn insert((array, data, index): (Value, Value, Value)) -> Result<Value, Error> {
+	match (array, index) {
+		(Value::Array(mut v), Value::Number(index)) => {
+			let mut index = index.as_int();
+			if index < 0 {
+				index = v.len() as i64 + index;
+			}
+			v.insert(index as usize, data);
```

Comment:
I think this will panic if index exceeds the length. We should probably return none or an error instead.

See: https://doc.rust-lang.org/std/vec/struct.Vec.html#method.insert

Reply:
oh ohk got it